portfolio/src/main/webapp/style.css

```diff
@@ -1,11 +1,34 @@
+body {
+    background-color: #937691; /* Light Purple */
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
-  color: #FEE9E1
+  font-family: Georgia;
+  color: #FEE9E1; /* Light Peach */
 }
 
-#random-fact-container {
-  margin-top: 20px;
+#content img {
+  width: 200px;
+  height: 200px;
+}
+
+h1, h2 {
+    background-color: #023C40; /* Dark Green */
+}
+
+#content button {
+    background-color: #023C40; /* Dark Green */
+    color: #FEE9E1; /* Light Peach */
```

Comment:
how did you choose these colors?

don't forget to verify the contrast is good between the text and the background! https://webaim.org/resources/contrastchecker/ can help (we aim for AA, there's an explanation at the bottom) You can also use the chrome dev tools to automate this: https://developers.google.com/web/tools/chrome-devtools/accessibility/reference#audits

Reply:
ooh noted thank you! the dark green and peach pass AA, but the dark pink wasn't dark enough, so I found a darker background color to use instead for the main page (#693561). I chose the colors from part of a color scheme that I made on https://coolors.co/